lib/cli/please.js

```diff
@@ -1,2 +1,2 @@
-require('please-update-dependencies')(module)
+// require('please-update-dependencies')(module)
```

Comment:
it should be either removed or explained why it is commented

Reply:
removed